src/commands/org/open/agent.ts

```diff
@@ -53,7 +53,7 @@ export class OrgOpenAgent extends OrgOpenCommandBase<OrgOpenOutput> {
     this.connection = this.org.getConnection(flags['api-version']);
 
     const [frontDoorUrl, retUrl] = await Promise.all([
-      buildFrontdoorUrl(this.org, this.connection),
+      buildFrontdoorUrl(this.org, this.connection, !(flags['url-only'] || this.jsonEnabled())),
```

Comment:
`org open agent` is in beta (safe to do breaking changes), let's make this one just use single-use URLs from start 👍🏼